### PR TITLE
DMS-19 added branding for Malawi

### PR DESCRIPTION
**Pull request by Manoj-nathwani**
https://fjelltopp.atlassian.net/browse/DMS-19

image

### CODE REVIEW

File: ckanext/dms/templates/header.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      padding: 3px;
+      border-radius: 3px;
+      font-size: 15px;
+    }
+    .no-hover-highlight:hover{
+      text-decoration: none !important;
+    }
+  </style>
+
+{% endblock %}
```
Comment (Manoj-nathwani) [inline ckanext/dms/templates/header.html:69]: This will be replaced tomorrow with another PR once I redesign how the logo displays on tablet/mobile devices

**Review by tomeksabala — APPROVED** [review]
